activity/covid19/covid19_schema

```diff
@@ -2,8 +2,9 @@
     "@context": [
         "https://raw.githubusercontent.com/ReproNim/reproschema/refactor-1.0/contexts/generic",
         {
-            "wellbeing": "https://raw.githubusercontent.com/ReproNim/reproschema/master/activities/Wellbeing/items/",
-            "nda": "https://raw.githubusercontent.com/ReproNim/reproschema/master/activities/NDA/items/"
+            "wellbeing": "https://raw.githubusercontent.com/ReproNim/reproschema-library/refactor-1.0/activities/Wellbeing/items/",
+            "nda": "https://raw.githubusercontent.com/ReproNim/reproschema-library/refactor-1.0/activities/NDA/items/",
+            "demo": "https://raw.githubusercontent.com/sanuann/reproschema-library/master/activities/demographics_and_background_information_v1/items/"
```

Comment:
should this point to your repo?

Reply:
it wasn't merged to repronim yet. changed it